### ISSUE TITLE
Add appveyor support

### ISSUE THREAD

**Issue by yarikoptic** [con/tinuous#11]
Per https://github.com/con/tinuous/issues/5#issuecomment-772536000

**Comment by jwodder** [comment]
@yarikoptic Do you know of any projects that have scheduled Appveyor builds?  I'm need to figure out how those are identified in the API.

**Comment by yarikoptic** [comment]
hm, I do not think we have those for datalad.  But you could just add that in appveyor setup for this repo

**Comment by jwodder** [comment]
@yarikoptic I tried to enable Appveyor (for #5) for this repository by adding the Appveyor GitHub App to the con org, but I got an error.  Can you successfully enable Appveyor for this repo?

**Comment by yarikoptic** [comment]
I have tried and tried but still fail to bring `con` to be visible within https://ci.appveyor.com/account/yarikoptic/projects/new ... may be need to give appveyor more time to recognize the change making me "public" within  https://github.com/orgs/con/people ... I will buzz you whenever I finally succeed.

**Comment by yarikoptic** [comment]
eh, I fell a victim of my own "greed".  I did give permissions to OpenNeuroscience organization (I minted before got `con`) but not `con`.  Now removed https://github.com/OpenNeuroscience to avoid any possible future confusion etc. Now there is https://ci.appveyor.com/project/yarikoptic/tinuous -- not sure how to make `con` "thing" on appveyor, but I think this should work - it is bound to this repo
